CONTRIBUTING.md

```diff
@@ -321,3 +321,38 @@ stripped with no prior warning or consent from the member in question.
 Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
 the community of this project.
+
+## Point Release Procedure
+
+*   3 Stages
+    *   **Strategy Discovery**
+        *   Develop long term priorities, strategy and roadmap for the SDK
+        *   Release committee not yet defined as there is already a roadmap that can be used for the time being
+    *   **Concept Approval**
+        *   Time bound period for Request for Comment (RFC) on Architectural Decision Records (ADR)
+        *   Release committee criteria
+            *   Participation in all or almost all ADR discussions
+            *   Are they already active contributors?
+                *   Are they constantly making substantial contributions to the project's codebase, review process, documentation or RFC?
+            *   Do they have a stake in the Cosmos SDK?
+                *   Are they a client of the SDK?
+                *   Are they “[giving back](https://www.debian.org/social_contract)” to the software?
+            *   Code owners need to maintain participation in a process
+                *   Delegate representation in case of vacation or absence
```

Comment:
Is this saying that code owners must be members of the ADR committee? or that they must just be present in meetings?